remi/monster.js

```diff
@@ -1,123 +1,122 @@
 // monster.js
 // ==========
 // for parsing text files
-const readline = require('readline');
-const fs = require('fs');
-const path = require('path');
+const readline = require('readline')
+const fs = require('fs')
 
 // store monsters in arrays
-const three_star_drops = [];
-const four_star_drops = [];
-const five_star_drops = [];
-const five_star_gods = [];
-const GFE = [];
+const threeStarDrops = []
+const fourStarDrops = []
+const fiveStarDrops = []
+const fiveStarGods = []
+const GFE = []
 
 /**
  * Roll a monster.
  * @return {url, name} The key-value pair of the roll
  */
-function rollMonster() {
-    const tier = Math.floor(Math.random() * 100) + 1;
-    let drop;
+function rollMonster () {
+  const tier = Math.floor(Math.random() * 100) + 1
+  let drop
 
-    console.log(tier);
-    if (tier <= 10) {
-        // 10% chance for 3*
-        drop = three_star_drops[Math.floor(Math.random() * three_star_drops.length)];
-    } else if (tier > 10 && tier <= 45) {
-        // 35% chance for 4* or 5* non-god
-        drop = (Math.random() < 0.5) ? four_star_drops[Math.floor(Math.random() * four_star_drops.length)]:five_star_drops[Math.floor(Math.random() * five_star_drops.length)];
-    } else if (tier > 45 && tier <= 90) {
-        // 45% chance for 5* god
-        drop = five_star_gods[Math.floor(Math.random() * five_star_gods.length)];
-    } else {
-        // 10% chance for GFE
-        drop = GFE[Math.floor(Math.random() * GFE.length)];
-    }
-    let roll = {
-        url: drop.url,
-        name: drop.name,
-        rarity: drop.rarity,
-    };
+  console.log(tier)
+  if (tier <= 10) {
+    // 10% chance for 3*
+    drop = threeStarDrops[Math.floor(Math.random() * threeStarDrops.length)]
+  } else if (tier > 10 && tier <= 45) {
+    // 35% chance for 4* or 5* non-god
+    drop = (Math.random() < 0.5) ? fourStarDrops[Math.floor(Math.random() * fourStarDrops.length)] : fiveStarDrops[Math.floor(Math.random() * fiveStarDrops.length)]
+  } else if (tier > 45 && tier <= 90) {
+    // 45% chance for 5* god
+    drop = fiveStarGods[Math.floor(Math.random() * fiveStarGods.length)]
+  } else {
+    // 10% chance for GFE
+    drop = GFE[Math.floor(Math.random() * GFE.length)]
+  }
+  const roll = {
+    url: drop.url,
+    name: drop.name,
+    rarity: drop.rarity
+  }
 
-    return roll;
+  return roll
 }
 
 // Read in the monsters, segregate, and store
-const threeStarFile = 'monsters/three_star_drops.txt';
-const fourStarFile = 'monsters/four_star_drops.txt';
-const fiveStarFile = 'monsters/five_star_drops.txt';
-const fiveStarGodsFile = 'monsters/five_star_gods.txt';
-const GFEFile = 'monsters/GFE.txt';
+const threeStarFile = 'monsters/threeStarDrops.txt'
```

Comment:
These are filenames in directory /monsters, please change these as well.

Reply:
nah dude change back the filenames, only the variable names matter